src/core/selection/selection.view.js

```diff
@@ -31,6 +38,25 @@ export default class SelectionView extends View {
 		});
 
 		model.selectionChanged.watch(e => {
+			if (!e || e.changes.hasOwnProperty('mode')){
+				timeoutApply(() => {
+
+					const newClassName = `${GRID_PREFIX}-select-${model.selection().mode}`;
+
+					this.markup.body.classList.add(newClassName);
```

Comment:
maybe it's better idea to add one class to markup,view? insead of body head foot?